content/README.md

```diff
@@ -0,0 +1,73 @@
+# Apache Beam (incubating) website
```

Comment:
Just pushed a very minor tweak to _config.yml to ignore the README (and Ruby bits)